### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by kelunik**

### CODE REVIEW

**Comment by lucaswerkmeister** [comment]
I wonder if it would be better to translate “unread” as „neu“ (literally “new”). It’s a slight change in meaning, but IMO „ungelesen“ (especially with another syllable „ungelesene“) feels a bit long and slightly awkward. WDYT?

File: de.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-	"Unread messages from %s": "Unread messages from %s",
-	"Unread messages from %s and %s": "Unread messages from %s and %s",
+	"Unread messages in %s": "Ungelesene Nachrichten in %s",
+	"Unread messages in %s and %s": "Ungelesene Nachrichten in %s and %s",
+	"Unread messages from %s": "Ungelesene Nachrichten von %s",
+	"Unread messages from %s and %s": "Ungelesene Nachrichten von %s und %s",
 	"Unread messages from %%s, %%s and one other": {
-		"one": "Unread messages from %%s, %%s and one other",
-		"other": "Unread messages from %%s, %%s and %d others"
+		"one": "Ungelesene Nachricht von %%s, %%s und einem anderen",
```
Comment (lucaswerkmeister) [inline de.js]: I wonder if there’s a gender-neutral phrasing? (The only trick I can come up with is „und 1 anderen“, which looks bad.)

Comment (kelunik) [inline de.js]: I'm not sure, maybe "einer anderen Person".

Comment (lucaswerkmeister) [inline de.js]: Oh, that’s good.

**Comment by cutandpastey** [comment]
Closing as https://github.com/gitterHQ/gitter-translations/pull/13/files seems more complete. @kelunik @lucaswerkmeister we would appreciate your feedback on the above PR however. Thanks.

**Comment by lucaswerkmeister** [comment]
Alright, I looked over it and left some feedback.

**Pull request closed without merging**